lib/puppet/parser/compiler.rb

```diff
@@ -79,7 +79,7 @@ def add_resource(scope, resource)
     if @current_app
       # We are in the process of pulling application components out that
       # apply to this node
-      Puppet.notice "Check #{resource}"
+      Puppet.notice _("Check %{resource}") % { resource: resource }
```

Comment:
I'm not sure how valuable this line and 39 above are for externalization... thoughts?